.circleci/config.yml

```
@@ -38,7 +38,7 @@ commands:
                 command: |
                   apk add github-cli --no-progress --repository=https://dl-cdn.alpinelinux.org/alpine/latest-stable/community/
                   <<# parameters.pr >>
-                  VERSION="pr-$(expr "${CIRCLE_PR_NUMBER:-${CIRCLE_PULL_REQUEST##*/}}" % 10)"
+                  VERSION="pr-$(( ${CIRCLE_PR_NUMBER:-${CIRCLE_PULL_REQUEST##*/}} % 10 ))"
```

With rebase, should now be:
```suggestion
                  VERSION="pr-$(( ${CIRCLE_PR_NUMBER:-${CIRCLE_PULL_REQUEST##*/}} % $NUM_PR_SITES ))"
```